### PR TITLE
Dot net string ends starts with

### PR DESCRIPTION
**Pull request by gunnargestsson**
Add functions: StartsWith and EndsWith

### CODE REVIEW

File: BaseApp/COD3007.TXT [hunk 1]
```diff
@@ -2,8 +2,9 @@ OBJECT Codeunit 3007 DotNet_String
 {
   OBJECT-PROPERTIES
   {
-    Date=;
-    Time=;
+    Date=23.07.18;
```
Comment (JesperSchulz) [inline BaseApp/COD3007.TXT]: Try to avoid unnecessary changes which distract from the code you'd like us to review, such as adding date/time or the LineStart below; both of which shouldn't be submitted to the repo ;-)

**Comment by gunnargestsson** [comment]
Cleaned
